geomesa-kafka/geomesa-kafka-09-utils/src/main/scala/org/locationtech/geomesa/kafka09/KafkaUtils.scala

```diff
@@ -0,0 +1,54 @@
+/***********************************************************************
+  * Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+  * All rights reserved. This program and the accompanying materials
+  * are made available under the terms of the Apache License, Version 2.0
+  * which accompanies this distribution and is available at
+  * http://www.opensource.org/licenses/apache2.0.php.
+  *************************************************************************/
```

Comment:
@ddseapy headers are still slightly off - they can't have leading spaces :/
Which I did to try to conform with the existing headers we had, but in retrospect that doesn't play well with auto-formatting of comments and is annoying.